src/googleAnalytics.js

```diff
@@ -0,0 +1,16 @@
+import isUndefined from 'lodash/isUndefined';
+import { trackCustomEvent } from 'gatsby-plugin-google-analytics';
+
+import { currentlyExecutingGitBranch } from './environment';
+
+const trackGoogleAnalyticsEvent = (eventParams) => {
+  if (!isUndefined(window.ga)) {
```

Comment:
you could check the existence of the acceptance cookie in here as well I think

Reply:
That would only prevent custom events from being tracked. There must be a better way to turn the whole `ga` object into a no-op. I'll open a separate issue.